src/API.js

```diff
@@ -0,0 +1,38 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xJG2mdwj5m9C9rOtT2IJ/scores/';
+const ul = document.getElementById('names');
+function displayScores(list) {
+  ul.textContent = '';
+  list.forEach((score) => {
+    const li = document.createElement('li');
+    const userName = document.createElement('span');
+    const userScore = document.createElement('span');
+    userName.innerText = `${score.user} :`;
+    userScore.innerText = score.score;
+    userName.className = 'span';
+    userScore.className = 'span';
+    li.append(userName, userScore);
+    ul.appendChild(li);
+  });
+}
```

Comment:
- [ ] I think it's better to use ES6 arrow function annotation instead of the old function syntax. it is mandatory to use ES6 features in this module. Arrow functions provide a simplified structure, allowing for a more concise way to write functions.